arches/app/views/resource.py

```diff
@@ -553,7 +533,7 @@ def get_instance_permissions(self, resource_instance):
             len(get_users_with_perms(resource_instance))
             + len(get_groups_with_perms(resource_instance))
         ) > 1
-        instance_creator = get_instance_creator(resource_instance)
+        instance_creator = resource_instance.get_instance_creator_and_edit_permissions()
```

Comment:
I get this is why `ResourceInstance` is being replaced with `Resource` in this file. However loading that proxy model may also cause some additional overhead and it might be worth rethinking the pattern

Reply:
I ended up moving these creator methods back into the base class, then I reverted those instances.